hedera-node/test-clients/src/main/resource/testSystemFiles/duplicated-operation.json

```diff
@@ -7,35 +7,37 @@
 		{
 		  "opsPerSec": 10000,
 		  "operations": [
-			"CryptoCreate",
-			"CryptoTransfer",
-			"CryptoUpdate",
-			"CryptoDelete",
-			"CryptoGetInfo",
-			"CryptoGetAccountRecords",
-			"CryptoApproveAllowance",
-			"CryptoDeleteAllowance",
-			"ConsensusCreateTopic",
-			"ConsensusSubmitMessage",
-			"ConsensusUpdateTopic",
-			"ConsensusDeleteTopic",
-			"ConsensusGetTopicInfo",
-			"TokenGetInfo",
-			"TokenGetNftInfo",
-			"TokenGetAccountNftInfos",
-			"ScheduleDelete",
-			"ScheduleGetInfo",
-			"FileGetContents",
-			"FileGetInfo",
-			"ContractUpdate",
-			"ContractDelete",
-			"ContractGetInfo",
-			"ContractGetBytecode",
-			"ContractGetRecords",
-			"ContractCallLocal",
-			"TransactionGetRecord",
-			"GetVersionInfo"
-		  ]
+				"CryptoCreate",
+				"CryptoTransfer",
+				"CryptoUpdate",
+				"CryptoDelete",
+				"CryptoGetInfo",
+				"CryptoGetAccountRecords",
+				"CryptoApproveAllowance",
+				"CryptoDeleteAllowance",
+				"ConsensusCreateTopic",
+				"ConsensusSubmitMessage",
+				"ConsensusUpdateTopic",
+				"ConsensusDeleteTopic",
+				"ConsensusGetTopicInfo",
+				"TokenGetInfo",
+				"TokenGetNftInfos",
+				"TokenGetNftInfo",
+				"TokenGetAccountNftInfos",
+				"ScheduleDelete",
+				"ScheduleGetInfo",
+				"FileGetContents",
+				"FileGetInfo",
+				"ContractUpdate",
+				"ContractDelete",
+				"ContractGetInfo",
+				"ContractGetBytecode",
+				"ContractGetRecords",
+				"ContractCallLocal",
+				"TransactionGetRecord",
+				"GetVersionInfo",
+				"UtilPrng"
```

Comment:
Same as `zero-ops-groups.json`. Adding a change so we have only a single error in the file(duplicated operation)

Reply:
Can you change the indentation? It is hard to compare the differences right now.